style.css

```diff
@@ -137,21 +201,23 @@ button {
   border-radius: 5px;
   background-color: white;
   padding: 10px;
+  width: fit-content;
 }
 
 .project button {
```

Comment:
- [ ] You should add styling to the buttons as suggested by previous reviewer according to the Figma design. In my opinion, you don't need to merge this branch to fix the issue. You can have something like this:
     ```
    .project button {
         border: 1px solid #7f8cff;
         background: #fff;
         color: #6070ff;
        cursor: pointer;
     }
     
      .project button:hover {
          background: #6070ff;
         color: #fff;
     }

      .project button:active {
          background: #2230d2;
         color: #fff;
      }
```- [ ]